src/resources/genomicsdb_export_config.proto

```diff
@@ -54,6 +54,14 @@ message SparkConfig {
   optional int64 query_block_size_margin = 2;
 }
 
+// This name should be changed  (eg AnnotationConfig) since AnnotationService is also a class in genomicsdb.h  
```

Comment:
the "AnnotationService" type was renamed to  "AnnotationSource" so this comment needs to be removed.